lang/LangPrimSource/PyrArrayPrimitives.cpp

```diff
@@ -29,6 +29,12 @@ Primitives for Arrays.
 #include "SC_InlineBinaryOp.h"
 #include "SC_Constants.h"
 #include <string.h>
+#include "sc_popen.h"
+#include "PyrUnixPrim.cpp"
```

Comment:
either that or forward-declare the functions you use ... don't include cpp files (unless you know what you are doing)